go.mod

```diff
@@ -2,4 +2,13 @@ module github.com/surrealdb/surrealdb.go
 
 go 1.18
 
-require github.com/gorilla/websocket v1.5.0
+require (
+	github.com/gorilla/websocket v1.5.0
+    github.com/stretchr/testify v1.8.0
```

Comment:
I'm not for the use of testify, as it is a very large dependency that most people only use for basics such as equality check and nil check, which can both easily be done with a few lines of code

I can help with the code if need be

Reply:
The rest of the code seems fine to me

Reply:
Is there another library that has similar things? Doing things like
`
if err != nil || data != "" {
    panic(err)
}
`
Seems clunky, and requires you to think of the inverse of what should happen, and panicking doesnt always tell you why particular things have failed or not. Mind you this is just the way I saw the other code do

Reply:
Absolutely, just create a function that does that for you !
You can look at how the require.NotNil and assert.Equal functions are implemented for inspiration
Testify is not a bad library per se, but it's very heavy
And as the creators of the language (https://go-proverbs.github.io/) say, a little copying is better than a little dependency

Reply:
Fair. Seems a bit tedious, but I'll see what @tobiemh thinks

Reply:
Would be best to keep dependencies to a minimum 👍if pos

Reply:
Sounds good. I pulled in the IsEqual and removed the bells and whistles. I still believe this is better than manual error checking, and should be done in the other tests, keeping the code more readable and short.